lib/tjbot.js

```diff
@@ -756,20 +730,20 @@ TJBot.prototype.see = function() {
 
     // capture 'this' context
     var self = this;
-    
+
     return new Promise(function(resolve, reject) {
         temp.open('tjbot', function(err, info) {
             if (err) {
                 reject("error: could not open temporary file for writing at path: " + info.path);
             }
-            
+
             self._captureImage(info.path).then(function(buffer) {
-                console.info("> sending image to Watson Visual Recognition");
+                winston.debug("sending image to Watson Visual Recognition");
                 var params = {
                     images_file: fs.createReadStream(info.path + ".jpg"),
                     threshold: self.configuration.see.confidenceThreshold.object
                 };
-                
+
                 self._visualRecognition.classify(params, function(err, response) {
```

Comment:
One more thing. Currently the see() method returns an array of text.
It previously returned the visual recognition response object and a text description.
I would vote that it returns both the response object and the text array similar to conversation.
Research being that users might want to specify a custom vision classifier and have access to both the response from their classifier and the default classifier. We should think about whether we want to support this.